mem-15-S.xml

```diff
@@ -1642,7 +1642,7 @@ Tätä sanaa ei käytetä lajittelujärjestykseen.[3] [AUTOTRANSLATED]</column>
       <column name="notes_ru">Это животное напоминает лошадь. [AUTOTRANSLATED]</column>
       <column name="notes_zh_HK">這種動物很像一匹馬。 [AUTOTRANSLATED]</column>
       <column name="notes_pt">Este animal se assemelha a um cavalo. [AUTOTRANSLATED]</column>
-      <column name="notes_fi">Tämä eläin muistuttaa hevosta. [AUTOTRANSLATED]</column>
+      <column name="notes_fi">-</column>
```

Comment:
Is there some reason this can't follow the same format as other languages, where "sark" is in the entry name and the description that it resembles a horse is in the notes? I guess this is fine as a workaround to suppress the notes, but will Finnish users be curious as to why there is a "-" in the notes?

If you want, I can make this way of suppressing the notes "official" by not displaying anything when the translation of notes into a language contains just "-".

Reply:
The way animals are defined in English is very inconsistent. For example, **gha'cher** is *goar-like creature*, **Qaj** is *type of animal, kradge*, **norgh** is *type of animal, norg (shark-like sea creature)*, and **targhoroj** is just *crab* (even though it is a Klingon animal).

I understand that this is because canonical sources don't have consistent form for definitions. For Finnish, there is no need to follow strictly canon definitions, so I've decided to always write "eräs X:ää muistuttava eläin, Y" meaning "a type of animal resembling X, Y".

It would be great if you make `-` suppress notes! This would solve problem with entries like this (and also some other entries where the note is unnecessary in Finnish, for example **ngob'at** had a redundant note).